support_test.go

```diff
@@ -14,6 +14,7 @@ func TestBase26(t *testing.T) {
 		26: "BA",
 		27: "BB",
 		52: "CA",
+		-1: "-B",
```

Comment:
Hmm.   Is that the right answer?
The numbers as letters are funny.  I want to start counting with "A" for 0.  Maybe "-A" is a better answer for -1 though.

Reply:
Okay, marking this as resolved (won't fix in this PR).